satpy/etc/composites/aws.yaml

```diff
@@ -0,0 +1,46 @@
+sensor_name: aws
```

Comment:
Looking at https://space.oscar.wmo.int/satellites/view/aws_pfm , the instrument is named MWR, so I guess the sensor (and the file) should also be `mwr(.yaml)`

Reply:
Yes, right. Sensor name is "MWR"